### PR TITLE
Fix case study links

### LINKED ISSUE #657: Homepage: some Featured Stories links are 404
The `read full case study` links for DataArt and Linkerd show 404 pages atm.

### PR DESCRIPTION
**Pull request by jhedev**
Closes #657

### CODE REVIEW

File: includes/case-studies.jade [hunk 1]
```diff
@@ -27,7 +27,7 @@
         .copy-wrapper
           h3.case-study-title linkerd + DC/OS: microservices in production made easy
           p.case-study-description linkerd is Buoyant’s open source traffic layer manager for microservices. If you’re running microservices on DC/OS, you can install the linkerd DC/OS package today to get programmable control over the flow of all RPC traffic throughout your application.
-          a(href='https://blog.buoyant.io/2016/04/19/linkerd-dcos-microservices-in-production-made-easy/' target="_blank").cta.cta--text.case-study-link Read full case study &rarr;
+          a(href='https://blog.buoyant.io/2016/04/19/linkerd-on-dc/os-microservices-in-production-made-easy/' target="_blank").cta.cta--text.case-study-link Read full case study &rarr;
```
Comment (karlkfi) [inline includes/case-studies.jade]: this is incorrect. the link was correct as-is

Comment (karlkfi) [inline includes/case-studies.jade]: actually, both work. the url is just weird with the / in there.

**Comment by karlkfi** [comment]
@mesosphere-ci: ok to test

File: includes/case-studies.jade (hunk 1 above)
Comment (jhedev) [inline includes/case-studies.jade]: You're right. Both links were corrected in the meantime. Feel free to close :)

**Comment by karlkfi** [comment]
ok to test

**Comment by karlkfi** [comment]
test this please

**Comment by karlkfi** [comment]
test this please

**Comment by karlkfi** [comment]
build triggers appear to be broken. bear with me.

**Comment by karlkfi** [comment]
oops! had the Commit comment hook set up instead of the Issue comment hook

**Comment by karlkfi** [comment]
ok to test

**Comment by karlkfi** [comment]
ok to test

**Comment by karlkfi** [comment]
and of course we run into a space issue on CI after last night's massive amount of builds.

**Comment by karlkfi** [comment]
retest this please

**Pull request merged**